src/main/java/org/moltimate/moltimatebackend/ScheduledTasks.java

```diff
@@ -0,0 +1,27 @@
+package org.moltimate.moltimatebackend;
+
+import lombok.extern.slf4j.Slf4j;
+import org.moltimate.moltimatebackend.catalyticsite.CatalyticSiteService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+/**
+ * A common location to store any scheduled tasks.
+ */
+@Component
+@Slf4j
+public class ScheduledTasks {
+
+    @Autowired
+    private CatalyticSiteService catalyticSiteService;
+
+    /**
+     * Update catalytic site database entries from the Catalytic Site Atlas every 24 hours.
+     */
+    @Scheduled(fixedDelay = 24 * 60 * 60 * 1000)
```

Comment:
this is every 24 hours, right?

Reply:
Yes. `fixedDelay` takes a milliseconds value, and will run the method on program start. Once the task finishes running, it will wait 24 hours to run again. This will work fine for now if we start doing long-term deployments. We should change this out for a cron expression (which is also supported) so that it runs at an off-peak time of the day.